.github/workflows/deploy.yml

```diff
@@ -10,14 +10,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7, 3.8 ]
+        python-version: [3.6, 3.7, 3.8]
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-      - name: Install dependencies
+      - name: Print environment values
+        run: |
+          cat $GITHUB_ENV
       - name: Update pip
```

Comment:
What does this fix? Was having an empty step causing a problem?

Reply:
Yes, this was the error from the last deploy attempt. 

![image](https://user-images.githubusercontent.com/11376379/128939671-fd069875-d000-4975-b1f8-79a1fd892c95.png)